Lib/pdb.py

```diff
@@ -444,9 +444,16 @@ def default(self, line):
         if line[:1] == '!': line = line[1:].strip()
         locals = self.curframe_locals
         globals = self.curframe.f_globals
+        readline = None
         try:
             if (code := codeop.compile_command(line + '\n', '<stdin>', 'single')) is None:
                 # Multi-line mode
+                try:
+                    import readline
```

Comment:
Maybe it's worth doing this just once at module level? I see we're already trying to import readline up in `Pdb.__init__`, so maybe we should just do this move once at module scope.